lib/core/opts.ts

```diff
@@ -31,4 +31,16 @@ export type Opts = {
    * Overrides the default execution id.
    */
   eid: string;
+
+  /**
+   * Test Only
+   * Probability of failure.
+   */
+  test?: number;
+
+  /**
+   * Test Only
+   * Random Seed.
+   */
+  testRandomSeed?: number;
```

Comment:
As discussed let's remove, I suspect that in the long run the `test` opt will become an object itself with a bunch of options, but for now let's keep it simple